### PR TITLE
MGMT-12200: Deprecate bump-ocp job

### PR DESCRIPTION
**Pull request by osherdp**
Already been added to https://gitlab.cee.redhat.com/assisted-installer-ops/assisted-installer-cicd, this removes the periodic OCP bump job.

A complete removal of the relevant workflow and step will have to wait for CRT approval because it's an existing workflow in cluster-bot.

/cc @adriengentil

### CODE REVIEW

File: ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh
Comment (adriengentil) [inline ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: why we need to keep it?

Comment (osherdp) [reply ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: just to prevent people adding functionality to the old script at openshift-assisted/assisted-installer-deployment
but alternatively we can just remove that script from there before this PR gets in and make sure merge this right after

Comment (adriengentil) [reply ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: ah ok, let's merge this now and do the proper cleanup later

**Review by adriengentil — APPROVED** [review]

File: ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh
Comment (osherdp) [reply ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: Implemented at https://github.com/openshift-assisted/assisted-installer-deployment/pull/307

**Comment by adriengentil** [comment]
/hold
@osherdp do you want to remove everything as part of this PR once https://github.com/openshift-assisted/assisted-installer-deployment/pull/307 is merged ?

**Comment by osherdp** [comment]
> /hold
> @osherdp do you want to remove everything as part of this PR once https://github.com/openshift-assisted/assisted-installer-deployment/pull/307 is merged ? 

I don't mind either way
So let's wait for that other PR (on assisted-installer-deployment) to merge and I'll change this PR

File: ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh
Comment (adriengentil) [reply ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: we can remove all the files in `ci-operator/step-registry/assisted/tools/bump-ocp/*`, no?

Comment (osherdp) [reply ci-operator/step-registry/assisted/tools/bump-ocp/assisted-tools-bump-ocp-commands.sh:9]: not yet. it's registered as a cluster-bot workflow and I'm not sure how fast we'll get the lgtm for it: https://github.com/openshift/release/blob/d8b442d080464217cf48037ddd75c3fc99499b7c/core-services/ci-chat-bot/workflows-config.yaml#L40
I'll follow up with another PR once it's getting in

**Review by adriengentil — APPROVED** [review]

**Comment by osherdp** [comment]
thank you!
/unhold

**Pull request merged**
